src/main/resources/values/values/strings.xml

```diff
@@ -119,51 +119,51 @@
     <string name="backup_create_sub">Create a backup from Tachidesk</string>
 
     <!-- General Settings -->
-    <string name="start_screen">Start Screen</string>
-    <string name="confirm_exit">Confirm Exit</string>
+    <string name="start_screen">Start screen</string>
+    <string name="confirm_exit">Confirm exit</string>
     <string name="language">Language</string>
-    <string name="language_system_default">System Default (%1$s)</string>
-    <string name="date_format">Date Format</string>
-    <string name="date_system_default">System Default</string>
+    <string name="language_system_default">System setting (%1$s)</string>
+    <string name="date_format">Date format</string>
+    <string name="date_system_default">System setting</string>
 
     <!-- Library Settings -->
-    <string name="show_all_category">Show all category</string>
+    <string name="show_all_category">Show all categories</string>
 
     <!-- Reader Settings -->
     <string name="reader_mode">Reader Mode</string>
     <string name="direction">Direction</string>
-    <string name="dir_down">Down</string>
+    <string name="dir_down">🠗</string>
```

Comment:
I'm a bit iffy on these, Maybe change them to Up to Down and Down to Up

Reply:
I tried going with arrows for LTR and RTL too.
These are awkward in other languages, because of how infrequently they are used.